export/index.template.html

```diff
@@ -399,12 +406,66 @@ <h2>Message log</h2>
         delete jwkPrivateCopy.d;
         jwkPrivateCopy.key_ops = ["verify"];
 
-        const publicKey =  await window.crypto.subtle
-          .importKey("jwk", jwkPrivateCopy, { name: "ECDSA", namedCurve: "P-256" }, true, ["verify"]);
+        const publicKey =  await window.crypto.subtle.importKey(
+          "jwk", 
+          {
+              kty: jwkPrivateCopy.kty,
+              crv: jwkPrivateCopy.crv,
+              x: padBase64Url(jwkPrivateCopy.x, 32),
+              y: padBase64Url(jwkPrivateCopy.y, 32),
+              ext: jwkPrivateCopy.ext,
+          }, 
+          { 
+              name: "ECDSA", 
+              namedCurve: "P-256" }, 
+          true, 
+          ["verify"]
+        );
+
         const buffer = await crypto.subtle.exportKey("raw", publicKey);
         return new Uint8Array(buffer)
       }
 
+    /**
+     * Decodes a base64url encoded string and pads the hex string correctly, encoding the result into a base64url encoded string
+     * @param {string} base64UrlString
+     * @param {number} length: optional number of bytes contained in the resulting string
+     * @return {string}
+     */
+    var padBase64Url = function(base64UrlString, length) {
+      var stringBuffer = base64urlDecode(base64UrlString);
+      var hexString = uint8arrayToHexString(stringBuffer)
+      // Add an extra 0 to the start of the string to get a valid hex string (even length)
+      // (e.g. 0x0123 instead of 0x123)
+      var hexString = hexString.padStart(Math.ceil(hexString.length/2)*2, 0)
```

Comment:
Tiny nit: I realized `padStart`'s second argument should be a string, so in this case `"0"`. We use this in a bunch of places and don't necessarily do it that way everywhere, but commenting as an FYI more than anything else